lib/state/chaintime.ts

```diff
@@ -0,0 +1,26 @@
+import { isRpcSdk } from "@zeitgeistpm/sdk-next";
+import { ChainTime } from "@zeitgeistpm/utility/dist/time";
+import { atom, getDefaultStore, useAtom } from "jotai";
+import { sdkAtom } from "lib/hooks/useSdkv2";
+import { Observable, Subscription } from "rxjs";
+
+export const chainTimeAtom = atom<ChainTime | false>(false);
+
+const store = getDefaultStore();
+
+let sub: Subscription;
+
+store.sub(sdkAtom, () => {
+  const sdk = store.get(sdkAtom);
+  if (sub) sub.unsubscribe();
```

Comment:
Why do we want to unsubscribe if the subscription already exists?

Reply:
Its in case we connect a new instance of the sdk to another endpoint.